moonbeam-types-bundle/index.ts

```diff
@@ -45,6 +44,14 @@ export const moonbeamDefinitions = {
         },
         ExtrinsicSignature: "EthereumSignature",
         RoundIndex: "u32",
+        Candidate: {
+          validator: "AccountId",
+          fee: "Perbill",
+          nominators: "OrderedSet<Bond<AccountId, Balance>>",
```

Comment:
Does bond also need to be defined separately? Or how does it know what the inner type is here?